llvm/lib/Target/AArch64/AArch64TargetTransformInfo.cpp

```diff
@@ -1630,6 +1630,36 @@ static std::optional<Instruction *> instCombineSVETBL(InstCombiner &IC,
   return IC.replaceInstUsesWith(II, VectorSplat);
 }
 
+static std::optional<Instruction *> instCombineSVEUzp1(InstCombiner &IC,
+                                                       IntrinsicInst &II) {
+  Value *A, *B;
+  Type *RetTy = II.getType();
+  constexpr Intrinsic::ID From_SV = Intrinsic::aarch64_sve_convert_from_svbool;
+  constexpr Intrinsic::ID To_SV = Intrinsic::aarch64_sve_convert_to_svbool;
```

Comment:
Not sure the coding standard is keen on `_` in variable names.  Also I read `SV` as "Scalable Vector", which doesn't mean much in the context.  What About `FromSVB` to solve both issues?